maintainers/maintainer-list.nix

```diff
@@ -8201,6 +8201,12 @@
       fingerprint = "1401 1B63 393D 16C1 AA9C  C521 8526 B757 4A53 6236";
     }];
   };
+  rowanG077 = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add rowanG077``.